riotfile.py

```diff
@@ -1731,9 +1731,29 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="unittest",
+            command="python -m pytest {cmdargs} tests/contrib/unittest_plugin/",
+            pkgs={"msgpack": latest},
+            env={"DD_CIVISIBILITY_AGENTLESS_ENABLED": "0"},
+            venvs=[
```

Comment:
Unless there's a clear reason not to, I recommend running these tests under all versions of Python from 2.7 to 3.11, this library's [list of supported versions](https://github.com/DataDog/dd-trace-py/blob/bf1b3c2594b1d1beed86ba0393314ac5d1faca35/setup.py#L601).

Reply:
Good catch, should be addressed in this new commit